### ISSUE TITLE
Crash when loading font on Japanese Windows 10

### ISSUE THREAD

**Issue by johachi** [python-pillow/Pillow#4468]
### What did you do?
Tried to draw text to an image using:

```py
ANNOTATION_FONT_SIZE = 20
fnt = ImageFont.truetype("ARIAL.TTF", ANNOTATION_FONT_SIZE)
text = "Test Text"
text_color = (64, 148, 224)
draw.text((50, 50), text, text_color, font=fnt)
```

### What did you expect to happen?
The text to be drawn

### What actually happened?
Application Crashed

```sh
Traceback (most recent call last):
  File "render_annotations.py", line 240, in <module>
    main(file_path)
  File "render_annotations.py", line 216, in main
    write_image_with_annotations(image_annotations, url, item["key"])
  File "render_annotations.py", line 182, in write_image_with_annotations
    newly_drawn_text_box = draw_annotation_to_image(annotation, image, drawn_text_boxes)
  File "render_annotations.py", line 154, in draw_annotation_to_image
    return draw_text(annotation, image, drawn_text_boxes)
  File "render_annotations.py", line 112, in draw_text
    fnt = ImageFont.truetype("ARIAL.TTF", ANNOTATION_FONT_SIZE)
  File "C:\Users\Man\AppData\Local\pypoetry\Cache\virtualenvs\qa-show-all-annotations-gvzPHQ0M-py3.7\lib\site-packages\PIL\ImageFont.py", line 642, in truetype
    return freetype(font)
  File "C:\Users\Man\AppData\Local\pypoetry\Cache\virtualenvs\qa-show-all-annotations-gvzPHQ0M-py3.7\lib\site-packages\PIL\ImageFont.py", line 639, in freetype
    return FreeTypeFont(font, size, index, encoding, layout_engine)
  File "C:\Users\Man\AppData\Local\pypoetry\Cache\virtualenvs\qa-show-all-annotations-gvzPHQ0M-py3.7\lib\site-packages\PIL\ImageFont.py", line 188, in __init__
    font, size, index, encoding, layout_engine=layout_engine
OSError: cannot open resource
PS C:\Users\Man\Documents\GitHub\crowd-ops\Projects\Add-Annotations> 
```

### What are your OS, Python and Pillow versions?

* OS: Windows 10 (Japanese)
* Python: 3.7.6
* Pillow: 7.0.0

```python
ANNOTATION_FONT_SIZE = 20
fnt = ImageFont.truetype("ARIAL.TTF", ANNOTATION_FONT_SIZE)
text = "Test Text"
text_color = (64, 148, 224)
draw.text((50, 50), text, text_color, font=fnt)
```

### Other

The font is installed (double-checked).

Saving the font in the same folder as the script and changing 
```py
fnt = ImageFont.truetype("ARIAL.TTF", ANNOTATION_FONT_SIZE)
```

to

```py
fnt = ImageFont.truetype("./ARIAL.TTF", ANNOTATION_FONT_SIZE)
```

is a viable work-around. 

Also see [sam210723/xrit-rx#9](https://github.com/sam210723/xrit-rx/issues/9)

**Comment by radarhere** [comment]
Would you be able to run the following script? This is code that I've put together to print the path of the font that Pillow is trying to access. Hopefully that should provide more insight.

```python
def printFontPath(path):
	from PIL import ImageFont
	freeType = ImageFont.FreeTypeFont

	class FreeTypeFont(freeType):
		def __init__(self, font=None, size=10, index=0, encoding="", layout_engine=None):
			if font != path:
				print(font)
			super().__init__(font, size, index, encoding, layout_engine)

	ImageFont.FreeTypeFont = FreeTypeFont
	try:
		ImageFont.truetype(path)
	except OSError:
		pass

printFontPath("ARIAL.TTF")
```

**Comment by johachi** [comment]
Thank you for the fast follow up.

I will run the script as soon as I get back to that computer.

**Comment by johachi** [comment]
It seems to be looking in the correct folder:

```sh
PS C:\Users\Man\Documents\GitHub> & C:/Users/Man/AppData/Local/Programs/Python/Python37-32/python.exe c:/Users/Man/Documents/GitHub/crowd-ops\Projects\Add-Annotations/font2.py
Note: Searches in C:\WINDOWS\fonts
PS C:\Users\Man\Documents\GitHub>
```

**Comment by radarhere** [comment]
Ah, but, my script should have also printed a path for the font. The fact that it didn't should mean that Pillow isn't finding 'ARIAL.TTF' in C:\WINDOWS\fonts.

What is the path to your installed ARIAL.TTF?

**Comment by johachi** [comment]
I somehow solved it.

Debugging
There is a `arial.ttf` file in `C:\WINDOWS\fonts`. That is also where windows say that the font is installed (picture 1). Also, in the script you gave me, I tried both `arial.ttf` and `ARIAL.ttf` but the result was the same for both.

I then opened the font by writing `C\WINDOWS\fonts\arial.ttf` in the command line. I don't know why, but the button for install was clickable (picture 2). The font installed and the path showing where the font is installed changed. (picture 3) to `C:\USERS\マヨ\APPDATA\LOCAL\MICROSOFT\WINDOWS\FONTS\ARIAL.TTF`. Pillow do now find the font if I write `arial.ttf` but not if I write `ARIAL.TTF`, **Also**, the script you gave me now prints the path where the font has been all along:

```sh
C:\Users\マヨ\Documents>python font2.py
Note: Searches in C:\WINDOWS\fonts
C:\WINDOWS\fonts\arial.ttf

C:\Users\マヨ\Documents>
```

Using the path in picture 3 gives the unicode error:

```sh
C:\Users\マヨ\Documents>python font2.py
  File "font2.py", line 24
    printFontPath("C:\USERS\マヨ\APPDATA\LOCAL\MICROSOFT\WINDOWS\FONTS\ARIAL.TTF")
                 ^
SyntaxError: (unicode error) 'unicodeescape' codec can't decode bytes in position 1-2: truncated \UXXXXXXXX escape

C:\Users\マヨ\Documents>
```

Important to note is that I could only get the "Install button" to be enabled by opening the file through the command line. Opening the preview through right click did not enable the install button.

I was able to replicate the problem/error by uninstalling the font making windows once again show that the font is installed in `C:\WINDOWS\FONTS\ARIAL.TTF`. I did this back and forth a few times to be sure that this was the cause. I then uninstalled it one last time. I then tried to access another font **bahnschrift.ttf** and that worked.  Very surprised I then switched back to `arial.ttf` and it still works, which surprised me even more.

I would like to write this of as a "misspelled font name" BUT I am 100% sure that spelled the font name correctly in lower case letters before I tried upper case. It also ran without a problem on mac machines. I was also able to recreate the problem a few times, so I just don't know what is going on... The reason I wrote this much is that I hope it will help anyone else who might run into the same problem in the future. I will try to reboot my PC after posting this. But if I can no longer replicate the error, then I do not know how I would be able to debug any further to find the root cause.

Thank you very much for the help @radarhere 

**Picture 1**
![Picture 1](https://user-images.githubusercontent.com/47516968/76621741-706c1f80-6573-11ea-9cde-040d6e5ea5fa.png)

**Picture 2**
![Picture 2](https://user-images.githubusercontent.com/47516968/76621791-942f6580-6573-11ea-9460-1076e3066ba0.png)


**Picture 3**
![Picture 3](https://user-images.githubusercontent.com/47516968/76621842-a4474500-6573-11ea-955f-7d9504109ed2.png)
